server-atlantis.yaml

```diff
@@ -9,7 +9,8 @@ workflows:
   terragrunt:
     plan:
       steps:
-      - run: tfenv install && tgenv install && terragrunt plan -no-color -out $PLANFILE
+      - run: tfenv install && tgenv install && ./import_resources.sh
```

Comment:
Does this work? You're running the import prior to terragrunt doing any initialization of the backend.tf. I was expecting the script to run as part of the terragrunt plan as a pre-trigger (or define the init step and have it do the install and the terragrunt init command, then have the plan run this script and the plan command).